buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijPlugin.kt

```diff
@@ -43,7 +61,7 @@ fun Project.configureIntellijPlugin() {
     pluginDescription.set(file("$rootDir/DESCRIPTION.html").readText())
 
     changeNotes.set(
-      "<h3>v${rootProject.version}</h3>\n\n${file("$rootDir/CHANGE-NOTES.html").readText()}"
+      "<h3>v${rootProject.version}</h3>\n\n${changelog.getUnreleased().toHTML()}"
```

Comment:
Hmm how does the result of `toHTML()` differ from what `getReleaseChangeLogForGithub` prints out?